### ISSUE TITLE
Improved SteamVR Binding Menu for Accessibility

### ISSUE THREAD

**Issue by Searith** [Yellow-Dog-Man/Resonite-Issues#1242]
### Is your feature request related to a problem? Please describe.

I use Oculus touch controllers but over time they have developed severe drift to the point where I have to use exclusively buttons for movement. When I booted up Resonite for the first time I had to change the bindings from default to fit my needs. 

It was made significantly harder by a few issues such as bindings being unlabeled other than what the button is and bindings being locked to the controller that the button is on, which leads to issues with specifically movement in joysticks. 

I have gotten everything to work with my setup, but it was very difficult and since I have a button on each controller for turning I had to disable "Allow Strafing" so both controllers can be used for turning.

### Describe the solution you'd like

Better labeling of SteamVR bindings, and allowing buttons from the other controller to be mapped.

### Describe alternatives you've considered

An in-game binding menu would also fix this, and potentially be even better than the SteamVR options.

**Comment by Frooxius** [comment]
Can you share screenshots of the SteamVR Bindings? They should generally be labeled directly with whatever the input is and you should be able to change them.

**Comment by ko-tengu** [comment]
Related probably, the Generic Controller bindings are labeled correctly, but get overridden by the Oculus Touch controller specific bindings, which are just the button labels.
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/89536622/9dedf0e8-9174-4c1f-9554-510e6ce483e9)

**Comment by Frooxius** [comment]
Yes. That is intentional. 

Generic is used when the controller is not any of the other ones. Other controllers are proxied through the controller-specific bindings.

They are labeled as the buttons, because we proxy those directly to Resonite, which then interprets them. If you want, you can adjust those bindings as well, it should not prevent you from doing so.

**Comment by ko-tengu** [comment]
Yeah, but that's the issue, I don't know what button_xa or joystick_click does, I would have a better idea of what Secondary or Radial Menu does.

The other issue with how it's bound is if I wanted to mirror the joysticks to have movement on my other hand, there's not a way to do that right now.

**Comment by Frooxius** [comment]
It does pretty much what it says - it notifies Resonite that particular button or input was pressed. 

Those are then interpreted on Resonite's end by its input system for actual in-game actions.

You can configure stuff there. E.g. swapping joysticks for movement can be done in the settings in Resonite.
